update_chroot

```diff
@@ -103,17 +103,17 @@ EMERGE_DEFAULT_OPTS="--verbose --oneshot"
 source "/etc/portage/make.conf.user"
 EOF
 
-sudo_clobber "/etc/portage/repos.conf/portage-stable.conf" <<EOF
+sudo_clobber "/etc/portage/repos.conf/gentoo-subset.conf" <<EOF
```

Comment:
Catalyst 4 sets them up this way, so it's easiest to follow suit.

Reply:
We're currently not deleting all the existing *.conf files though. We could do, but I think that would be impolite, especially given that we've been talking about custom overlays recently.

Reply:
With the churn happening in this area, maybe deleting these conf files is the sane approach. /etc is part of the container image (same with /board/<xxx>/etc) so user modifications get lost by design.

Custom overlays will require a proper design and integration with scripts - perhaps the user will add an overlay in `scripts/repos/....` and then this will be taken into account when generating the files, but either way - the configs are managed by scripts.